### ISSUE TITLE
Small UI issues in web app

### ISSUE THREAD

**Issue by tveulemans** [clipto-pro/Desktop#29]
Hello,

First of all: thanks for a great app!

I have come across two (very) small UI issues in the web app:

1. In Chrome I sometimes see a white rectangle on the bottom part of the right hand panel. This happens when the browser is maximized (and sometimes when it is not). I do not observe the same behavior in Edge.
2. When in editing mode there is a vertical navigation bar that appears on the right even if the note is very small (and there should therefore be no vertical navigation required). This happens in both Chrome and Edge.

Kind regards,
Thomas

Screenshot for issue 1
![cliptopro_white rectangle](https://user-images.githubusercontent.com/42804752/70135973-5e810380-168b-11ea-8f6a-fc4afe436b8e.png)

Screenshot for issue 2
![cliptopro_navigation bar in editing view](https://user-images.githubusercontent.com/42804752/70135972-5de86d00-168b-11ea-9889-bd8cae0d4d93.png)

**Comment by atrashler** [comment]
Hi! Have reproduced it. Thanks!

**Comment by atrashler** [comment]
Please check it. The version of the Web version - 1.7.6

**Comment by tveulemans** [comment]
That was fast!!!

Both issues are solved as far as I can see (I did not test extensively :-)).

Still one very small comment (related to issue 2): even though the note is short - and the navigation bar has now disappeared - I can still make the text of the note go up a little bit by scrolling with the mouse scroll wheel. This feels a bit counter-intuitive.

That being said, I'm going to close the issue (given that the two initial items have been resolved).

Once again: super impressed by the speed! And thanks for the app!
